### PR TITLE
Kmikawa/service principles

### PR DESCRIPTION
**Pull request by kmikawa**
This PR supports service principles.

## Description

- Updated a lib to generate access token.
- Updated the documentation to better reflect our support service principles.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: config/config.js [hunk 1]
```diff
@@ -6,19 +6,25 @@ NOTICE: Adobe permits you to use, modify, and distribute this file in
 accordance with the terms of the Adobe license agreement accompanying
 it.
 */
-const { awsConfig } = require('./aws')
+const fs = require('fs')
+const path = require('path')
 const { adobeConfig } = require('./adobe')
+
+let awsConfig
+if (fs.existsSync(path.join(__dirname, 'aws.js'))) {
```
Comment (nbelofastow) [inline config/config.js:14]: I wanna double check the logic here - users may or may not provide aws.js. May not, if they have no intention to run our samples with AWS S3 support or use our signing API. Is my understanding correct?

File: src/sample/psapi/11_getDocumentManifest.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ accordance with the terms of the Adobe license agreement accompanying
 it.
 */
 
-const awsLib = require('../../lib/awsLib')
+// const awsLib = require('../../lib/awsLib')
```
Comment (nbelofastow) [inline src/sample/psapi/11_getDocumentManifest.js:10]: Is this comment really necessary?

Comment (khound) [reply src/sample/psapi/11_getDocumentManifest.js:10]: Should cleanup.

File: config/config.js (hunk 1 above)
Comment (kmikawa) [reply config/config.js:14]: Yes, that's correct.

**Comment by khound** [comment]
👍

**Review by khound — APPROVED** [review]

**Pull request merged**